launch-script.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+export TMPDIR=$XDG_CACHE_HOME/tmp
```

Comment:
Why do you need persistent tmpdir? It's better to use `TMPDIR=$XDG_RUNTIME_DIR/app/$FLATPAK_ID` which is tmpfs